nearestColor.js

```diff
@@ -373,7 +382,7 @@
     '#808'  // v
   ]);
 
-  nearestColor.VERSION = '0.4.0';
```

Comment:
:(

Sorry I didn't notice this!

Reply:
no worries I bumped that up as well. Not sure why we need this though, since there is a NPM package version.